### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by jhass**

### CODE REVIEW

File: app/views/pages/_contributors.html.slim [hunk 1]
```diff
@@ -9,7 +9,14 @@ markdown:
...
   * [atmosx](https://github.com/atmosx)
+  * [baweaver](https://github.com/baweaver)
+  * [havenwood](https://github.com/havenwood)
   * [jhass](https://github.com/jhass)
   * [jheg](https://github.com/jheg)
+  * [ktzsid10](https://github.com/ktzsid10)
+  * [mstudov](https://github.com/mstudov)
+  * [sevenseacat](https://github.com/sevenseacat)
+  * [SkyBirdSoar](https://github.com/SkyBirdSoar)
+  * [SkyBirdSoar](https://github.com/SkyBirdSoar)
```
Comment (adam12) [inline app/views/pages/_contributors.html.slim]: Dup?

File: app/views/pages/_irc_staff.html.slim [hunk 2]
```diff
@@ -14,23 +20,28 @@ ul
...
-  li jhass (jhass)
-  li ljarvis (ljarvis)
-  li miah (miah)
-  li Mon_Ouie (Mon_Ouie)
-  li seanstickle (seanstickle)
-  li slyphon (slyphon)
-  li smathy (smathy)
-  li workmad3 (workmad3)
-  li zenspider (zenspider)
+  li banisterfiend (banisterfiend) (Freenode)
```
Comment (adam12) [inline app/views/pages/_irc_staff.html.slim:24]: Add me?

File: app/views/pages/_links.html.slim [hunk 3]
```diff
@@ -76,30 +70,25 @@ ul
     a href="https://www.railstutorial.org/book"
       | Ruby on Rails Tutorial
   li
-    a href="http://mislav.uniqpath.com/poignant-guide/book/chapter-1.html"
+    a href="http://poignant.guide/"
```
Comment (havenwood) [inline app/views/pages/_links.html.slim]: Looks like TLS works for the new link https://poignant.guide/

File: app/views/pages/_irc_staff.html.slim (hunk 2 above)
Comment (jhass) [reply app/views/pages/_irc_staff.html.slim:24]: I wanted to do it but lost track of while checking the status of all the others. Sorry :see_no_evil: :sweat_smile:

**Review by adam12 — APPROVED** [review]

**Pull request merged**